docs/modules/execa.md

```diff
@@ -0,0 +1,13 @@
+# Execa
+
+When you don't need [additional features](https://github.com/sindresorhus/execa/#features) that `execa` provides such as [string templating](https://github.com/sindresorhus/execa/blob/main/docs/execution.md#template-string-syntax), [inter-process communication](https://github.com/sindresorhus/execa/#exchange-messages), and [debugging features](https://github.com/sindresorhus/execa/?tab=readme-ov-file#detailed-error),  you can replace it with tinier alternatives below.
+
+# Alternatives
```

Comment:
we can possibly add:

- [zurk](https://github.com/webpod/zurk)
- [nanoexec](https://github.com/fabiospampinato/nanoexec)

let me know what you think

Reply:
From what I've read from `zurk` is that it's a form of experiment for `zx` and presumably be removed once `zx` has moved to v8. Is it OK for us to include temporary stuff like `zurk`.

As for `nanoexec`, I'll be adding that.

Reply:
if thats true im happy to leave zurk out until zx figures out what they're doing with it 👍